src/main/api/2/studio-api-2.yaml

```diff
@@ -4230,11 +4230,61 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /content/{siteId}:
+    get:
+      tags:
+        - content
+      summary: Get content for a path
+      description: 'Required permission "content_read"'
+      operationId: getContent
+      parameters:
+        - name: siteId
+          in: path
+          description: The site ID
+          required: true
+          schema:
+            type: string
+        - name: path
+          in: query
+          description: path of the content
+          required: true
+          schema:
+            type: string
+        - name: commitId
```

Comment:
@phuongnq please also indicate the default value for commitId

Reply:
@phuongnq you can also use the `default` key in the parameter schema:
```
schema:
  type: string
  default: HEAD
```

Reply:
I updated to use the schema default value. Thank you!